poly_commit/src/orion/simd_field_agg_impl.rs

```diff
@@ -83,12 +83,9 @@ where
         return false;
     }
 
-    if !orion_mt_verify(
-        vk,
-        &query_indices,
-        &proof.query_openings[..query_num],
-        &roots[0],
-    ) {
+    if izip!(proof.query_openings.chunks(query_num), &roots)
+        .any(|(paths, r)| !orion_mt_verify(vk, &query_indices, paths, r))
+    {
```

Comment:
this is a fixing to prior impl - supposing only root of MPI machine verifies, then eventually it should check all the MT openings, I only took care of the first party, i.e., root's MT opening, should just verify all of them